dotCMS/src/main/java/com/dotcms/business/bytebuddy/WrapInTransactionAdvice.java

```diff
@@ -0,0 +1,118 @@
+package com.dotcms.business.bytebuddy;
+
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.github.rjeschke.txtmark.Run;
+import net.bytebuddy.asm.Advice;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import java.lang.reflect.Method;
+import java.sql.Connection;
+
+public class WrapInTransactionAdvice {
```

Comment:
I am not sure if this advicer is handling the externalize=true

Reply:
Create a new annotation would be ok

Reply:
I this something we need at the moment for the system to work or can we add a task for this?